lib/gcloud/translate/translation.rb

```diff
@@ -91,8 +91,8 @@ def detected?
       ##
       # @private New Translation from a TranslationsListResponse object as
       # defined by the Google API Client object.
-      def self.from_response resp, text, to, from
-        res = text.zip(Array(resp.data["translations"])).map do |origin, gapi|
+      def self.from_gapi_list resp, text, to, from
```

Comment:
The issue here is that `Translate` has responsibility for decoding both the collection and the member. There is already a `from_gapi` for the member. What should I name the `from_gapi` for the collection? I don't really like `from_gapi_list`...